### DISCUSSION TITLE
Status Line

### DISCUSSION THREAD

**Discussion by korbosoft** [contour-terminal/contour#1057] [Q&A] [answered]
Is there any way to get rid of this? 
![image](https://user-images.githubusercontent.com/47117026/222994076-72e845e1-f779-40a9-8b2f-2d7a4b0f92f3.png)

**Comment by christianparpart** [comment, marked as answer]
Hey @Korbaux. this is fixed in latest master. How did you install Contour?

Seems like you are using zsh, can it be? It's making use of some terminfo entries (`tsl` / `fsl`) that it should not. If you installed from latest master, just update, otherwise, we're very soon about to create a new bugfix release. :)

Related PR: #1060.
